wurst/math/Maths.wurst

```diff
@@ -35,116 +36,42 @@ public function getSlopeAngle( real z1, real z2, real dist ) returns angle
-	return max(max(a, b), c)
-
-/** returns the bigger number of the four parameters */
-public function max(int a, int b, int c, int d) returns int
-	return max(max(a, b, c), d)
-
-/** returns the bigger number of the five parameters */
-public function max(int a, int b, int c, int d, int e) returns int
-	return max(max(a, b, c, d), e)
-
-/** returns the bigger number of the six parameters */
-public function max(int a, int b, int c, int d, int e, int f) returns int
-	return max(max(a, b, c, d, e), f)
-
-/** returns the bigger number of the two parameters */
-public function max(real x, real y) returns real
-	real r
-	if x > y
-		r = x
-	else
-		r = y
-	return r
-
-/** returns the bigger number of the three parameters */
-public function max(real a, real b, real c) returns real
-	return max(max(a, b), c)
-
-/** returns the bigger number of the four parameters */
-public function max(real a, real b, real c, real d) returns real
-	return max(max(a, b, c), d)
-
-/** returns the bigger number of the five parameters */
-public function max(real a, real b, real c, real d, real e) returns real
-	return max(max(a, b, c, d), e)
-
-/** returns the bigger number of the six parameters */
-public function max(real a, real b, real c, real d, real e, real f) returns real
-	return max(max(a, b, c, d, e), f)
-
-/** returns the smaller number of the two parameters */
-public function min(int x, int y) returns int
-	int r
-	if x < y
-		r = x
-	else
-		r = y
-	return r
-
-/** returns the smaller number of the three parameters */
-public function min(int a, int b, int c) returns int
-	return min(min(a, b), c)
-
-/** returns the smaller number of the four parameters */
-public function min(int a, int b, int c, int d) returns int
-	return min(min(a, b, c), d)
-
-/** returns the smaller number of the five parameters */
-public function min(int a, int b, int c, int d, int e) returns int
-	return min(min(a, b, c, d), e)
-
-/** returns the smaller number of the six parameters */
-public function min(int a, int b, int c, int d, int e, int f) returns int
-	return min(min(a, b, c, d, e), f)
-
-/** returns the smaller number of the two parameters */		
-public function min(real x, real y) returns real
-	real r
-	if x < y
-		r = x
-	else
-		r = y
-	return r
-	
-/** returns the smaller number of the three parameters */
-public function min(real a, real b, real c) returns real
-	return min(min(a, b), c)
-
-/** returns the smaller number of the four parameters */
-public function min(real a, real b, real c, real d) returns real
-	return min(min(a, b, c), d)
-
-/** returns the smaller number of the five parameters */
-public function min(real a, real b, real c, real d, real e) returns real
-	return min(min(a, b, c, d), e)
-
-/** returns the smaller number of the six parameters */
-public function min(real a, real b, real c, real d, real e, real f) returns real
-	return min(min(a, b, c, d, e), f)
-
+/** returns the bigger number of parameters */
```

Comment:
The language used in these doc comments is not quite correct. I guess you want something like:

```
/** Return the largest value in the set provided as arguments.

    Example: max(6, 4, -12)  // Returns 6.
*/
```